### DISCUSSION TITLE
Connection broken: IncompleteRead(505 bytes read, 7 more expected

### DISCUSSION THREAD

**Discussion by wlc002** [acheong08/ChatGPT#913] [General]
('Connection broken: IncompleteRead(505 bytes read, 7 more )', IncompleteRead(505 bytes read, 7 more expected))

why this happened sometime?

**Comment by wlc002** [comment]
also this error occurred
"Connection broken: InvalidChunkLength(got length b'', 0 bytes read)", InvalidChunkLength(got length b'', 0 bytes read))

**Reply by acheong08** [reply]
Internet errors cannot be fixed on my end

**Reply by wlc002** [reply]
ok if it's openai's reason,i have no idea
